diff --git a/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js b/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js
index 6ee58a4..c9ebab4 100644
--- a/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js
+++ b/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js
@@ -120,6 +120,7 @@ function createRollupOptions(options, dependencies, context, packageJson, source
                     tsconfigOverride: {
                         compilerOptions: createTsCompilerOptions(config, dependencies, options),
                     },
+                    clean: true,
                 }),
             shouldBundleTypes &&
                 (0, type_definitions_1.typeDefinitions)({
