diff --git a/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js b/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js
index e7e92ae..59998b2 100644
--- a/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js
+++ b/node_modules/@nx/rollup/src/executors/rollup/rollup.impl.js
@@ -126,6 +126,7 @@ function createRollupOptions(options, dependencies, context, packageJson, source
                     tsconfigOverride: {
                         compilerOptions: createTsCompilerOptions(config, dependencies, options),
                     },
+                    clean: true,
                 }),
             shouldBundleTypes &&
                 (0, type_definitions_1.typeDefinitions)({
