diff --git a/node_modules/nx/bin/nx.js b/node_modules/nx/bin/nx.js
index 5dbbef3..ba592ed 100755
--- a/node_modules/nx/bin/nx.js
+++ b/node_modules/nx/bin/nx.js
@@ -36,7 +36,7 @@ function main() {
     }
     else {
         if (workspace && workspace.type === 'nx') {
-            require('v8-compile-cache');
+            
         }
         // polyfill rxjs observable to avoid issues with multiple version of Observable installed in node_modules
         // https://twitter.com/BenLesh/status/1192478226385428483?s=20
