diff --git a/node_modules/path2d/dist/index.cjs b/node_modules/path2d/dist/index.cjs
index 15d3e01..a207a48 100644
--- a/node_modules/path2d/dist/index.cjs
+++ b/node_modules/path2d/dist/index.cjs
@@ -494,9 +494,20 @@ function roundRect(x, y, width, height, radii = 0) {
 function applyPath2DToCanvasRenderingContext(CanvasRenderingContext2D) {
   if (!CanvasRenderingContext2D)
     return;
+  const cClip = CanvasRenderingContext2D.prototype.clip;
   const cFill = CanvasRenderingContext2D.prototype.fill;
   const cStroke = CanvasRenderingContext2D.prototype.stroke;
   const cIsPointInPath = CanvasRenderingContext2D.prototype.isPointInPath;
+  CanvasRenderingContext2D.prototype.clip = function clip(...args) {
+    if (args[0] instanceof Path2D) {
+      const path = args[0];
+      const fillRule2 = args[1] || "nonzero";
+      buildPath(this, path.commands);
+      return cClip.apply(this, [fillRule2]);
+    }
+    const fillRule = args[0] || "nonzero";
+    return cClip.apply(this, [fillRule]);
+  };
   CanvasRenderingContext2D.prototype.fill = function fill(...args) {
     if (args[0] instanceof Path2D) {
       const path = args[0];
diff --git a/node_modules/path2d/dist/index.js b/node_modules/path2d/dist/index.js
index ddf4eb8..acb0c44 100644
--- a/node_modules/path2d/dist/index.js
+++ b/node_modules/path2d/dist/index.js
@@ -462,9 +462,20 @@ function roundRect(x, y, width, height, radii = 0) {
 function applyPath2DToCanvasRenderingContext(CanvasRenderingContext2D) {
   if (!CanvasRenderingContext2D)
     return;
+  const cClip = CanvasRenderingContext2D.prototype.clip;
   const cFill = CanvasRenderingContext2D.prototype.fill;
   const cStroke = CanvasRenderingContext2D.prototype.stroke;
   const cIsPointInPath = CanvasRenderingContext2D.prototype.isPointInPath;
+  CanvasRenderingContext2D.prototype.clip = function clip(...args) {
+    if (args[0] instanceof Path2D) {
+      const path = args[0];
+      const fillRule2 = args[1] || "nonzero";
+      buildPath(this, path.commands);
+      return cClip.apply(this, [fillRule2]);
+    }
+    const fillRule = args[0] || "nonzero";
+    return cClip.apply(this, [fillRule]);
+  };
   CanvasRenderingContext2D.prototype.fill = function fill(...args) {
     if (args[0] instanceof Path2D) {
       const path = args[0];
