diff --git a/node_modules/prettier/index.js b/node_modules/prettier/index.js
index 75a972b..b7b9be4 100644
--- a/node_modules/prettier/index.js
+++ b/node_modules/prettier/index.js
@@ -34116,7 +34116,7 @@ var require_Markdown = __commonJS2({
       codemirrorMode: "gfm",
       codemirrorMimeType: "text/x-gfm",
       wrap: true,
-      extensions: [".md", ".livemd", ".markdown", ".mdown", ".mdwn", ".mdx", ".mkd", ".mkdn", ".mkdown", ".ronn", ".scd", ".workbook"],
+      extensions: [".md", ".livemd", ".markdown", ".mdown", ".mdwn", ".mdx", ".mkd", ".mkdn", ".mkdown", ".ronn", ".scd", ".workbook",".mdoc"],
       filenames: ["contents.lr"],
       tmScope: "source.gfm",
       languageId: 222
